pz_be_services/routers/v1/user_router.py

```
@@ -100,6 +97,90 @@ def login(user: UserLogin, db: Session = Depends(get_db)):
+GITHUB_CLIENT_ID = EnvironmentVariables.GITHUB_CLIENT_ID
+GITHUB_CLIENT_SECRET = EnvironmentVariables.GITHUB_CLIENT_SECRET
+CLIENT_REDIRECT_URI = EnvironmentVariables.CLIENT_REDIRECT_URI
+
+GITHUB_AUTHORIZE_URL = EnvironmentVariables.GITHUB_AUTHORIZE_URL
+GITHUB_TOKEN_URL = EnvironmentVariables.GITHUB_TOKEN_URL
+GITHUB_USER_API = EnvironmentVariables.GITHUB_USER_API
+
+FRONTEND_REDIRECT_URL = EnvironmentVariables.FRONTEND_USER_URL
+
+@router.get("/login/github")
+def login():
```

Ruff (0.12.2)

Redefinition of unused `login` from line 59

(F811)